apps/nextjs-x-chain/src/components/CCTPTransfer.tsx

```diff
@@ -109,7 +121,7 @@ export function CCTPTransfer({
     if (!originWalletDetails) return;
     const fetchWalletUsdcBalance = async () => {
       const balance = await crossChainCore.getWalletUSDCBalance(
```

Comment:
USDC balance on source chain?

Reply:
are you suggesting rename? or asking what this function does?